modules/android/vsandroid_vcxproj.lua

```diff
@@ -45,10 +64,10 @@
 
 		return elements
 	end)
-	
+
 	premake.override(vc2010.elements, "globalsCondition", function (oldfn, prj, cfg)
 		local elements = oldfn(prj, cfg)
-		
+
```

Comment:
Can you revert these whitespace changes?

Reply:
These are caused by the editorconfig from the project so it seemed normal to me, but sure.